src/net/sourceforge/kolmafia/KoLAdventure.java

```diff
@@ -1498,8 +1513,8 @@ public boolean canAdventure() {
       return switch (this.adventureNumber) {
         case AdventurePool.DRUNKEN_STUPOR -> KoLCharacter.isFallingDown();
         case AdventurePool.SSPD_STUPOR -> today.contains(holiday)
-            && KoLCharacter.getInebriety() >= 26;
-        default -> (holiday == null) ? true : today.contains(holiday);
+            || today.equals("Drunksgiving") && KoLCharacter.getInebriety() >= 26;
```

Comment:
We used to have (in KoLmafia.java):

```
      } else if (!holiday.contains("St. Sneaky Pete's Day") && !holiday.contains("Drunksgiving")) {
        KoLmafia.updateDisplay(MafiaState.ERROR, "You are too drunk to continue.");
        return;
      } else if (KoLCharacter.getInebriety() <= 25) {
        KoLmafia.updateDisplay(MafiaState.ERROR, "You are not drunk enough to continue.");
        return;
      }
```

You do the first check and print the error message in preValidateAdventure.
You are doing the second check here. I think you should print the error message.

Reply:
> Perhaps I should let you submit this, and then un-disable some tests.

Yeah that was my plan. I didn't write these tests, I just observef that they were implicitly disables and instead made them explicitly disabled so we're less likely to forget about them!